libs/vue-plugin/src/builders/browser/builder.ts

```diff
@@ -0,0 +1,98 @@
+import {
+  BuilderContext,
+  BuilderOutput,
+  createBuilder
+} from '@angular-devkit/architect';
+import { normalizeAssetPatterns } from '@angular-devkit/build-angular/src/utils';
+import { join, normalize, virtualFs } from '@angular-devkit/core';
+import { NodeJsSyncHost } from '@angular-devkit/core/node';
+import { from, Observable, of } from 'rxjs';
+import { map, switchMap } from 'rxjs/operators';
+import { BrowserBuilderSchema } from './schema';
+import { getProjectRoot, getProjectSourceRoot } from '../../utils';
+import {
+  modifyCachePaths,
+  modifyCopyAssets,
+  modifyEntryPoint,
+  modifyIndexHtmlPath,
+  modifyTsConfigPaths
+} from '../../webpack';
+
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const Service = require('@vue/cli-service/lib/Service');
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const { resolvePkg } = require('@vue/cli-shared-utils/lib/pkg');
+
+export function runBuilder(
+  options: BrowserBuilderSchema,
+  context: BuilderContext
+): Observable<BuilderOutput> {
+  if (!options.outputPath) {
+    context.logger.error('Invalid options, "outputPath" is required.');
+    return of({ success: false });
+  }
+
+  async function setup(): Promise<{
+    projectRoot: string;
+    inlineOptions;
+  }> {
+    const projectRoot = await getProjectRoot(context);
+    const projectSourceRoot = await getProjectSourceRoot(context);
+    //https://github.com/angular/angular-cli/blob/v9.1.0/packages/angular_devkit/build_angular/src/browser/index.ts#L574
+    const normalizedAssetPatterns = normalizeAssetPatterns(
```

Comment:
Have you seen [this](https://github.com/nrwl/nx/blob/a95cba314484465d618b5bc7c0faacd885e8720d/packages/web/src/utils/normalize.ts#L76)? I don't know if it accomplishes the same thing, but it would remove the dependency on `@angular-devkit/build-angular`. I noticed `@nrwl/web` doesn't have that dependency as well.

Reply:
I hadn't. I think it's meant to accomplish the same thing. Using it wouldn't save a dependency, it would replace `@angular-devkit/build-angular` with `@nrwl/web`. I don't see a reason to make this change now, but perhaps in the future. It's good to be aware of.